rubix-bookkeeper/pom.xml

```diff
@@ -61,6 +61,13 @@
             <scope>test</scope>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
We can remove this if we use bookkeeper readdata to download data.

Reply:
removed